docs/api/auto-updater.md

```diff
@@ -68,7 +68,7 @@ Emitted when there is an error while updating.
 
 ### Event: 'checking-for-update'
 
-Emitted when checking if an update has started.
+Emitted when checking if an update is available has started.
```

Comment:
Not trying to bikeshed too much, but I think this wording is still a little confusing. What do you think of?:
```suggestion
Emitted when checking for an available update has started.
```

I like clarifying the check is for available updates and not for updating itself though 🙂

Reply:
Works for me! I see what you mean. Thanks for the comment!